_client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html

```diff
@@ -0,0 +1,43 @@
+<link rel="import" href="../../bower_components/paper-dialog/paper-action-dialog.html">
+<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/core-item/core-item.html">
+
+<polymer-element name="iwish-dialog" atributes="">
+	<template>
+		<section on-tap="{{toggleDialog}}">
+  		<button>dialog</button>
+  		<paper-action-dialog backdrop autoclosedisabled heading="filter">
+	  		<div>
+	    		<paper-radio-group selected="someFresh">
+	    			<paper-radio-button name="someFresh" label="some fresh"></paper-radio-button>
+	    			<paper-radio-button name="mostVoted" label="most voted"></paper-radio-button>
+	    			<paper-radio-button name="mostCommented" label="most commented"></paper-radio-button>
+	    		</paper-radio-group>
+	  		</div>
+	  		<div>
+	  			<core-item icon="settings" label="tag"></core-item>
+	  		</div>
+	  		<div>
+	  			<core-item icon="settings" label="word or phrase"></core-item>
+	  			<paper-input></paper-input>
+	  		</div>
+    		<paper-button affirmative raised class="colored">Done</paper-button>
+  		</paper-action-dialog>
+  	</section>	
+	</template>
+	<script>
+		Polymer('iwish-dialog', {
+      toggleDialog: function(e) {
+        if (e.target.localName != 'button') {
+          return;
+        }
+        var d = e.target.nextElementSibling;
+        if (!d) {
+          return;
+        }
+        d.toggle();
+      }
+		})
```

Comment:
I guess we need to configure jshint to look through html's as well. Add ```;``` please + ```'use strict'``` (*yes, i will add those to my components as well)